scheduler/factories.go

```diff
@@ -52,6 +53,19 @@ func factoryTaskMessage(t models.Task) (res []byte, err error) {
 			return
 		}
 
+	case 4: //TRACEROUTE
+		testTracerouteTask, er := traceroute.NewTaskFromModel(t)
+		if er != nil {
+			err = er
+			return
+		}
+
+		res, err = json.Marshal(testTracerouteTask) //nolint
```

Comment:
Why do we have `nolint` here @arthureld? It should be avoided in general and applied only for false-positives.